controller/server/view/network_details_page.ml

```diff
@@ -252,6 +218,21 @@ let html service =
     |> sexp_of_t
     |> Sexplib.Sexp.to_string_hum
   in
+  let disconnect_button =
+    form
+      ~a:[ a_action ("/network/" ^ service.id ^ "/remove")
+      ; a_method `Post
+      ; a_class [ "d-Network__Forget" ]
+      ; Unsafe.string_attrib "is" "disable-after-submit"
+      ]
+      [ input
+          ~a:[ a_input_type `Submit
+          ; a_class [ "d-Button" ]
+          ; a_value "Forget"
```

Comment:
I would be more comfortable with `Disconnect`. Does iOS use this naming ?

Reply:
It seems to be widely used:

- iOS
- Android
- [Connman GTK](https://github.com/jgke/connman-gtk/blob/b72c6ab3bb19c07325c8e659902b046daa23c506/po/connman-gtk.pot#L597-L599)
- [Windows](https://www.lifewire.com/forget-network-in-windows-11-5208278)

I think "Forget" is better than "Disconnect" because it implies that this isn't just a temporary disconnect and all settings will be forgotten.

Reply:
OK.